lib/qa/authorities/linked_data/config/search_config.rb

```diff
@@ -5,17 +5,20 @@
 module Qa::Authorities
   module LinkedData
     class SearchConfig
-      attr_reader :prefixes
+      attr_reader :prefixes, :full_config, :search_config
+      private :full_config, :search_config
+
+      delegate :authority_name, to: :full_config
 
       # @param [Hash] config the search portion of the config
-      def initialize(config, prefixes = {})
+      # @param [Hash<Symbol><String>] prefixes URL map of prefixes to use with ldpaths
+      # @param [Qa::Authorities::LinkedData::Config] full_config the full linked data configuration that the passed in search config is part of
+      def initialize(config, prefixes = {}, full_config = nil)
```

Comment:
passing `full_conf` to search config allows access to the full authority config for information like `authority_name`.